webpack.common.config.ts

```diff
@@ -48,7 +48,11 @@ module.exports = {
     extensions: ['.ts', '.js', '.json', '.html', '.svg', '.png'],
     alias: {
       '@angular/upgrade/static': (
-        '@angular/upgrade/bundles/upgrade-static.umd.js')
+        '@angular/upgrade/bundles/upgrade-static.umd.js'),
+      // This is needed because in app.constants.ts we need to import
+      // assets/consants.ts. We can't directly write import 'constants'
```

Comment:
constants is misspelled.

Below, "in" is misspelled.

Please review **all** your comments properly, in all the files for this PR, before you do the next push, and fix them, otherwise reviewers might end up only finding them one by one.

Reply:
Done. :sweat_smile: